Lab1/app/src/pro/java/com/examplePro/Lab1/ScreenFragment.java

```diff
@@ -0,0 +1,132 @@
+package com.examplePro.Lab1;
```

Comment:
This fragment is very similar to the fragment from the 'free' version. Can you use inheritance to avoid code duplication?